migrations/00001_initial/index.sql

```diff
@@ -0,0 +1,16 @@
+DROP SCHEMA IF EXISTS emails CASCADE;
+CREATE SCHEMA emails;
+
+DROP TABLE IF EXISTS emails.signer_emails CASCADE;
+CREATE table emails.signer_emails
+(
+    id                   SERIAL PRIMARY KEY,
+    chain_id             int                      NOT NULL,
+    email_address        text                     NOT NULL,
+    safe_address         character varying(42)    NOT NULL,
+    signer               character varying(42)    NOT NULL,
+    verified             boolean                  NOT NULL DEFAULT false,
+    verification_code    text,
+    verification_sent_on timestamp with time zone NOT NULL DEFAULT now(),
```

Comment:
As I commented in internal discussions, I'd consider not setting this default at the database level. I'm just leaving a comment here as well as a reminder.

Reply:
I agree. Removed the default for the `verification_sent_on`. This provides more flexibility on the domain layer on when this field should be set.